docs/_documentations/eclipse-firstproject.md

```diff
@@ -43,14 +43,13 @@ This icon launches your web browser and displays your application.
 
 Congratulations, you have now created, built and run your first Codewind project using Codewind on your local machine.
 
-
 # Next Steps
 
-For next steps, choose from the following:
+Choose from the following:
 
-* If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
+**Getting started with Codewind - next steps** Continue to instructions for [Making a Code change with Eclipse](./eclipse-codechange.html) to see the auto-build-and-run feature in action.
 
-1. Deploy Codewind to your cloud if not already done so by you or a sysadmin/DevOps engineer. See tutorial [Deploying Codewind Remotely](./remote-deploying-codewind.html).
-2. Connect your Codewind extension of your local desktop IDE to Codewind in your cloud by following the tutorial [Using Codewind Remotely](./remote-codewind-overview.html).
+**Installing Codewind for remote use** If you intend to use Codewind [remotely](./remote-codewind-overview.html), you have now completed the pre-requisite step of installing Codewind on your local machine and can proceed to:
 
- * To continue with this [Using Codewind Locally](./local-codewind-overview.html) tutorial, continue to the next step [Making a Code change with Eclipse](./eclipse-codechange.html) to see the auto-build-and-run feature in action.
\ No newline at end of file
+1. Deploy Codewind to your cloud if not already done so by you or a sysadmin/DevOps engineer. See tutorial [Deploying Codewind Remotely](./remote-deploying-codewind.html).
+2. Connect your Codewind extension of your local desktop IDE to Codewind in your cloud by following the tutorial [Using Codewind Remotely](./remote-codewind-overview.html).
```

Comment:
I really like the simplicity of the "Next steps" format that we currently have in place, such as here: https://www.eclipse.org/codewind/eclipse-firstproject.html

Could we omit the "Choose from the following:" statement and streamline the links in a bulleted list? For example, maybe something like this?

```
## Next steps
- **[Making a code change with Eclipse](./eclipse-codechange.html):** See the auto-build-and-run feature in action.
- **[Overview: Using Codewind remotely](./remote-codewind-overview.html):** If you intend to use Codewind remotely, you can proceed to one of the following steps:
   - **[Deploying Codewind Remotely](./remote-deploying-codewind.html):** Deploy Codewind to your cloud.
   - **[Using Codewind Remotely](./remote-codewind-overview.html):** Connect the Codewind extension of your local desktop IDE to Codewind in your cloud.
```

- For the last two links on lines 54 and 55, assuming it is not mandatory to complete them in sequential order, we should use an unordered list instead of an ordered list, as mentioned in the wiki: https://github.com/eclipse/codewind-docs/wiki/Writing-style-guidelines#lists
- Also, I have changed the heading size to the `##` size. The `#` size heading is good for the main title of the page, as mentioned in our wiki: https://github.com/eclipse/codewind-docs/wiki/Writing-style-guidelines#heading-and-titles

Reply:
The Next Steps section has been enhanced because the different user flows need to be covered. This has all resulted from the User Flow Design Work that has been done, with orientation being added.

Reply:
It is mandatory for the user to complete the steps of 'deploying' and 'using' in that order.